gno.land/cmd/gnoland/config_get_test.go

```diff
@@ -0,0 +1,650 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/gnolang/gno/tm2/pkg/bft/config"
+	"github.com/gnolang/gno/tm2/pkg/commands"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfig_Get_Invalid(t *testing.T) {
+	t.Parallel()
+
+	// Create the command
+	cmd := newRootCmd(commands.NewTestIO())
+	args := []string{
+		"config",
+		"get",
+		"--config-path",
+		"",
+	}
+
+	// Run the command
+	cmdErr := cmd.ParseAndRun(context.Background(), args)
+	assert.ErrorContains(t, cmdErr, "unable to load config")
+}
+
+// testSetCase outlines the single test case for config get
+type testGetCase struct {
+	name     string
+	field    string
+	verifyFn func(*config.Config, string)
+}
+
+// verifyGetTestTableCommon is the common test table
+// verification for config set test cases
+func verifyGetTestTableCommon(t *testing.T, testTable []testGetCase) {
+	t.Helper()
+
+	for _, testCase := range testTable {
+		testCase := testCase
+
+		t.Run(testCase.name, func(t *testing.T) {
+			t.Parallel()
+
+			// Setup the test config
+			path := initializeTestConfig(t)
+			args := []string{
+				"config",
+				"get",
+				"--config-path",
+				path,
+			}
+
+			// Create the command IO
+			mockOut := bytes.NewBufferString("")
```

Comment:
```suggestion
			mockOut := new(bytes.Buffer)
```

Reply:
I laughed at the thought you were hesitant to nit on this, but it got the better of you :)

Fixed, for all of these buffer inits in the codebase:

[0adf819](https://github.com/gnolang/gno/pull/1605/commits/0adf819730ce096f31e089e1660d7af63b36caa6)